package.json

```diff
@@ -38,27 +45,22 @@
     "madrone",
     "vue"
   ],
-  "dependencies": {},
   "devDependencies": {
-    "@types/jest": "^29.5.4",
     "@types/lodash": "^4.14.197",
-    "@typescript-eslint/eslint-plugin": "^6.5.0",
-    "@typescript-eslint/parser": "^6.5.0",
-    "esbuild": "^0.19.2",
-    "eslint": "^8.48.0",
+    "@typescript-eslint/eslint-plugin": "^6.7.3",
+    "@typescript-eslint/parser": "^6.7.3",
+    "eslint": "^8.50.0",
     "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-import": "^2.28.1",
-    "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-unicorn": "^48.0.1",
-    "jest": "^29.6.4",
-    "jest-environment-jsdom": "^29.6.4",
+    "happy-dom": "^12.6.0",
     "lodash": "4.17.21",
     "prettier": "^3.0.3",
```

Comment:
looks like we don't need prettier to be installed any more, so should remove the dependency 😄

Reply:
good catch! missed that one 👍